lib/cfa/grub2/default.rb

```diff
@@ -139,8 +139,14 @@ def terminal=(values)
         generic_set("GRUB_TERMINAL", values.join(" "))
       end
 
+      # Sets GRUB_SERIAL_COMMAND option
+      #
+      # Updates GRUB_SERIAL_COMMAND with given value, also enables serial
+      # console in GRUB_TERMINAL
+      #
+      # @param value [String] value for GRUB_SERIAL_COMMAND
       def serial_console=(value)
-        self.terminal = :serial
+        self.terminal = self.terminal | [:serial]
```

Comment:
this is problematic when self.terminal is nil ( can be ). In such case it sets terminal to `true` which we really do not want. so I want something like `(self.terminal || []) | [:serial]`